website/content/partials/components/builder-docker-pull.mdx

```diff
@@ -36,7 +36,7 @@ These parameters are used in the [`use` stanza](/docs/waypoint-hcl/use) for this
 
 The image to pull.
 
-This should NOT include the tag (the value following the ':' in a Docker image). Use `tag` to define the image tag.
+This should NOT include the tag (the value following the ':' in a Docker image). Use `tag` to define the  image tag.
```

Comment:
Not related to this PR but looks like this has an extra space defined:

https://github.com/hashicorp/waypoint/blob/99d1d67aca31349d4edf84198916e6769ed996aa/builtin/docker/pull/builder.go#L91

Reply:
Looks like the space was removed in `builtin/docker/pull/builder.go` so maybe this needs generated still?